soccer/src/soccer/strategy/agent/position/defense.cpp

```diff
@@ -33,6 +35,14 @@ Defense::State Defense::update_state() {
             walling_robots_ = {(u_int8_t)robot_id_};
             break;
         case WALLING:
+            // If a wall is already full,
+            // Remove the robot with the highest ID from a wall
+            // and make them a marker instead.
+            if (this->robot_id_ == *max_element(walling_robots_.begin(), walling_robots_.end()) &&
```

Comment:
it's really minor but can we flip the order of this &&? just logically, the size() check is more likely to be false, and also faster to execute